Revisa los errores flake8:

```
./mrp_dump/models/mrp_production.py:42:80: E501 line too long (81 > 79 characters)
./mrp_dump/__openerp__.py:31:1: W293 blank line contains whitespace
```

--- a/mrp_dump/__openerp__.py
+++ b/mrp_dump/__openerp__.py
@@ -28,7 +28,7 @@
     "description": """
     This module provides an association between the bulk product and
     the way it must be packaged at the end of the bulk product manufacturing.
-    
+
     Particularly suitable for chemical industries and
     others that produce liquid finished products.
     """,

--- a/mrp_dump/models/mrp_production.py
+++ b/mrp_dump/models/mrp_production.py
@@ -39,7 +39,8 @@ class DownloadOperation(models.Model):
     def _calculate_weight(self):
         self.fill = self.ul.ul_qty * self.qty
 
-    product = fields.Many2one('product.product', string='Product', required=True)
+    product = fields.Many2one('product.product', string='Product',
+                              required=True)
     operation = fields.Many2one('mrp.production')
     ref = fields.Many2one('stock.warehouse', string='Warehouse')
     qty = fields.Integer(string="QTY")

